client/layout/masterbar/minimal.jsx

```diff
@@ -5,22 +5,34 @@
  */
 import React from 'react';
 import Masterbar from './masterbar';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Item from './item';
 
-const MasterbarMinimal = ( { url } ) => (
+const MasterbarMinimal = ( { url, loggedOut, children } ) => (
```

Comment:
Yeah, we didn't switch to using the minimal masterbar in checkout after the a/b test ended, so that can be removed.